src/boat_simulator/tests/unit/nodes/physics_engine/test_fluids.py

```diff
@@ -1,9 +1,69 @@
 """Tests classes and functions in boat_simulator/nodes/physics_engine/fluids.py"""
 
+import numpy as np
+import pytest
+from boat_simulator.common.generators import ConstantGenerator, GaussianGenerator, VectorGenerator, MVGaussianGenerator
+from boat_simulator.nodes.physics_engine.fluid_generation import FluidGenerator
 
 class TestFluidGenerator:
-    pass
 
+    @pytest.mark.parametrize("vector", [
+            (np.array([1, 0])),
+            (np.array([0, 1])),
+            (np.array([1, 0])),
+    ])
+    def test_velocity_constant(self, vector):
+        vector_generator = ConstantGenerator(constant=vector)
+        fluid_generator = FluidGenerator(generator=vector_generator)
+        first_fluid_vector = fluid_generator.velocity
+        assert np.all(first_fluid_vector == vector)
```

Comment:
It might be beneficial to include an additional assertion after calling `fluid_generator.next()`, comparing the newly generated vector with the expected vector. This will help ensure that the `next()` function behaves as expected and doesn't cause unexpected changes.